packages/reports/addon/components/report-actions/multiple-format-export.js

```diff
@@ -70,20 +82,37 @@ export default class MultipleFormatExport extends Component {
   /**
    * @property {Array} exportFormats - A list of export formats
    */
-  @computed('csvHref', 'pdfHref')
+  @computed('csvHref', 'pdfHref', 'supportedFileTypes')
   get exportFormats() {
-    return [
+    const { supportedFileTypes } = this;
+
+    const exportFormats = [
       {
         type: 'CSV',
         href: this.csvHref,
         icon: 'file-text-o'
-      },
-      {
-        type: 'PDF',
-        href: this.pdfHref,
-        icon: 'file-pdf-o'
       }
     ];
+
+    if (Array.isArray(supportedFileTypes)) {
+      if (supportedFileTypes.includes('pdf') || supportedFileTypes.includes('PDF')) {
+        exportFormats.push({
+          type: 'PDF',
+          href: this.pdfHref,
+          icon: 'file-pdf-o'
+        });
+      }
+
+      if (supportedFileTypes.includes('png') || supportedFileTypes.includes('PNG')) {
+        exportFormats.push({
+          type: 'PNG',
+          href: this.pdfHref.then(href => `${href}&fileType=png`),
```

Comment:
Can we change `this.pdfHref` to something like `this.exportHref` or `this.rasterizeHref`?